packages/types/src/indexed-db.ts

```diff
@@ -93,27 +91,27 @@ export interface PenumbraDb extends DBSchema {
   // ======= Json serialized values =======
   // Allows wasm crate to directly deserialize
   ASSETS: {
-    key: Base64Str; // Jsonified<DenomMetadata['penumbraAssetId']['inner']>
-    value: Jsonified<DenomMetadata>;
+    key: Base64Str; // DenomMetadata['penumbraAssetId']['inner']
+    value: JsonValue;
```

Comment:
Why change these to `JsonValue`? Isn't e.g., `Jsonified<DenomMetadata>` more useful now that we have a proper `Jsonified` type?

Reply:
the upgrade has complex downstream effects :) for now it's just stubbed out